frontend_lib/src/component/ShareDownload/NewShareDownload.jsx

```diff
@@ -19,7 +19,7 @@ export class NewShareDownload extends React.Component {
     }
   }
 
-  handleTogglePopoverMultipleEmails = () => {
+  popoverMultipleEmailsToggle = () => {
```

Comment:
Wouldn't `toggleMultipleEmailsPopover` be clearer? (verb first).

Reply:
Indeed.